api/wan/read

```diff
@@ -74,7 +74,7 @@ if($cmd eq 'providers') {
     $ret->{'configuration'}{'interfaces'} = \@interfaces;
     my @tmp = split(",",$cdb->get_prop('firewall','CheckIP'));
     $ret->{'configuration'}{'multiwan'}{'CheckIP'} = \@tmp;
-    foreach (qw(MaxNumberPacketLoss MaxPercentPacketLoss NotifyWan PingInterval WanMode)) {
+    foreach (qw(MaxNumberPacketLoss MaxPercentPacketLoss NotifyWan NotifyWanFrom NotifyWanTo PingInterval WanMode)) {
```

Comment:
Question: shall we consider root/EmailAddress and root/SenderAddress as default values for the NotifyWanFrom and NotifyWanTo props?

Reply:
Of course!

The NotifyWanFrom NotifyWanTo  should be completely ignored by cockpit since have been deprecated.